### ISSUE TITLE
Debian UEFI build

### ISSUE THREAD

**Issue by stefangweichinger** [tylert/packer-build#41]
A general question:

Did you lately test `base-uefi.pkr.hcl` ? I try to get that working (Debian 11.3) with VirtualBox on my Fedora system and see various issues. The `boot_command` seems not to work anymore etc.

Just asking if that build works for you, @tylert  ... thanks!

**Comment by tylert** [comment]
I don't currently have a nice way to test these templates at the moment as I'm currently working on mostly ARM machines these days.

If you're stuck for a working image, there's also pre-built ones like https://cloud.debian.org/images/cloud/bullseye/daily/20220531-1032/ available too.

**Comment by stefangweichinger** [comment]
OK, understood. Thanks for the link to the cloud images. I have a working packer setup for Debian 11.3 non-uefi already, so I am good with that. I just played with creating a UEFI-variant and noticed that it's not so easy to add that. The `boot_command` needs to be different, the partitioning, etc. I tried to pick from your setup but wasn't successful so far. But it's no "have-to" here, just hacking around.
